mir-ci/mir_ci/fixtures/servers.py

```diff
@@ -90,3 +92,18 @@ def mir_demo_server():
 @server
 def miriway():
     return snap("miriway", channel="stable", classic=True)
+
+
+@server(
+    ServerCap.ALL
+    ^ (ServerCap.FLOATING_WINDOWS | ServerCap.DRAG_AND_DROP | ServerCap.DISPLAY_CONFIG | ServerCap.SCREENCOPY)
```

Comment:
It can certainly do floating windows and dnd :)

```suggestion
    ^ (ServerCap.DISPLAY_CONFIG | ServerCap.SCREENCOPY)
```

We may need to tweak the tests (like `xfail` them), but we should lie about the capabilities of the server.